test/core/tsi/s2a/s2a_test_data.h

```diff
@@ -31,6 +31,20 @@ std::vector<uint8_t> test_message_2 = {8};
 std::vector<uint8_t> test_message_3 = {46,  98, 101, 255, 213, 156, 15,  100,
                                        126, 45, 130, 239, 209, 13,  156, 89};
 
+std::vector<uint8_t> padded_zeros_message = {'1', '2', '3', '4', '5', '6'};
```

Comment:
Should these be zeroes? It could be that the name is a bit confusing.

Reply:
No, the message is correct. I could change it to `message_encrypted_with_padded_zeros`?

Reply:
Yes that would be clearer. Thanks!

Reply:
Done.